examples/nats-req/__main__.py

```diff
@@ -72,15 +75,25 @@ async def reconnected_cb():
         print(e)
         show_usage_and_die()
 
+    async def cb(msg):
+        await nc.publish(msg.reply, b'reply to hello world')
+
+    await nc.subscribe(args.subject, cb=cb)
+
     print(f"Connected to NATS at {nc.connected_url.netloc}...")
     msg = await nc.request(args.subject, args.data.encode())
     subject = msg.subject
     reply = msg.reply
     data = msg.data.decode()
-    print("Received a message on '{subject} {reply}': {data}".format(
-        subject=subject, reply=reply, data=data))
+    print(
+        "Received a message on '{subject} {reply}': {data}".format(
```

Comment:
nit: f-string

Reply:
I do agree fstrings would be good, I'm going to tackle those in a followup pr as this one is going to be large enough as is